package.json

```diff
@@ -94,38 +94,39 @@
     "@commercetools/jest-enzyme-matchers": "1.1.2",
     "@commitlint/cli": "7.5.2",
     "@commitlint/config-conventional": "7.5.0",
-    "@emotion/babel-preset-css-prop": "10.0.7",
+    "@emotion/babel-preset-css-prop": "10.0.9",
     "@percy/puppeteer": "1.0.5",
-    "@storybook/addon-actions": "5.0.3",
-    "@storybook/addon-info": "5.0.3",
-    "@storybook/addon-knobs": "5.0.3",
-    "@storybook/addon-links": "5.0.3",
-    "@storybook/addon-options": "5.0.3",
-    "@storybook/addon-storysource": "5.0.3",
-    "@storybook/react": "5.0.3",
-    "@storybook/theming": "5.0.3",
+    "@storybook/addon-actions": "5.0.5",
+    "@storybook/addon-info": "5.0.5",
+    "@storybook/addon-knobs": "5.0.5",
+    "@storybook/addon-links": "5.0.5",
+    "@storybook/addon-options": "5.0.5",
+    "@storybook/addon-storysource": "5.0.5",
+    "@storybook/react": "5.0.5",
+    "@storybook/theming": "5.0.5",
```

Comment:
seems to be a breaking change here as it fails without having `emotion-theming` installed 🤦‍♂️